gulpfile.js/deploy.js

```diff
@@ -35,7 +35,7 @@ const TIMESTAMP = Date.now();
 // We tag docker images by the current git commit SHA + TIMESTAMP,
 // this makes it easy to identify and reproduce builds.
 // Pass a specific tag via commandline using `gulp updateStart --tag ABCDE...`
-const TAG = argv.tag || `${require('@lib/utils/git').version}-${TIMESTAMP}`;
+const TAG = argv.tag || `${require('@lib/utils/git').version()}-${TIMESTAMP}`;
```

Comment:
this would resolve as a `string || Promise`, no?

Reply:
good catch :)